src/app/components/country-select/country-select.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, OnInit } from '@angular/core';
+import { RegionSelectService } from 'src/app/services/region-select.service';
```

Comment:
Minor point, but should we call this component a region-select, or rename the service to deal with countries just to be consistent?